src/Analyzers/Core/CodeFixes/GenerateConstructor/AbstractGenerateConstructorService.State.cs

```diff
@@ -92,17 +92,17 @@ private async Task<bool> TryInitializeAsync(
         {
             if (_service.IsConstructorInitializerGeneration(_document, node, cancellationToken))
             {
-                if (!await TryInitializeConstructorInitializerGenerationAsync(node, cancellationToken).ConfigureAwait(false))
+                if (!TryInitializeConstructorInitializerGeneration(node, cancellationToken))
```

Comment:
The core api these were all calling into is SymbolFinder.FindSourceDefinitionAsync.  This method is just a wrapper around the synchronous SymbolFinder.FindSourceDefinition method (but part of the public api, so we can't remove it).  Updating all internal callers to call the synchronous version caused a lot of code to become synchronous.